scalableminds/webknossos
# Use new zip.js version to allow zip64 uploads

I wasn't able to upload the recent negative_float test dataset, getting the message that the zip file was invalid, even after re-zipping. I investigated and found out that the 4 year old repackaged version of zip.js we used doesn't support zip64 (which is used if the files > 4 GB are zipped). In the meantime, zip.js supports webpack and also supports zip64 which is why I decided to use the official version instead. Review ignoring white space changes.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Upload a dataset and check that the uploader still is able to find wkw and nml files inside the zip

------
(Please delete unneeded items, merge only when none are left open)
